eureka_ml_insights/data_utils/transform.py

```diff
@@ -302,3 +304,45 @@ def transform(self, df: pd.DataFrame, encoding="cl100k_base") -> pd.DataFrame:
             token_count_column = f"{column}_token_count"
             df[token_count_column] = token_count
         return df
+
+@dataclass
+class MajorityVoteTransform:
```

Comment:
Every time you write a new transform class, please write a test for it in data_tests where the rest of transform tests live.

Reply:
Added a test